docs/HelmChart.md

```diff
@@ -0,0 +1,317 @@
+the organization, public sources (like open source) as well as third party
+vendors. There are different collectors for the different types of locations
+that GUAC can watch: files, storage buckets, git repositories, etc.
+
+Collectors are able to be configured to use a CollectSub service to know which
+data sources are of interest. For example, a git collector may subscribe to the
+CollectSub service to know which git repositories it should get its data from.
+This is a way in which one can, at a glance get an idea of what data sources the
+instance of GUAC is looking at!
+
+Collectors and certifiers then take the documents and pass them on to the
+ingestor via [NATS](https://nats.io/).
+
+#### Certifiers
+
+Another class of GUAC collector components are certifiers. Certifiers run
+outside the server, but are not part of the ingestion pipeline. They watch the
+server for new nodes in the server, and then try to add additional information
+attached to those nodes.
+
+For example, the [OSV](https://ossf.github.io/osv-schema/) certifier will watch
+GUAC for new packages, then try to discover OSV vulnerabilities for those
+packages. If any vulnerabilities are found, the certifier will attach a
+"CertifyVuln" node to the package that signifies that the package is connected
+to the OSV vulnerability.
+
+#### Ingestor
+
+Ingestors take in documents and parse them into the GUAC data model/ontology.
+The process extracts meaning from documents and translates to a common reasoning
+model (GUAC ontology). In the process, it also finds identifiers of interest in
+which it passes to the CollectSub service to request additional information for.
+
+Today, GUAC can understand multiple data formats such as: SPDX, CycloneDX, SLSA,
+etc. The ingestor listens for documents to parse via [NATS](https://nats.io/),
+and talks to the Assembler via a GraphQL API.
+
+#### Assembler
+
+The assembler takes the parsed GUAC ontology objects from the Ingestor and
+creates entries within a database which is used as a source of truth for GUAC
+queries.
+
+The assembler exposes a set of GraphQL mutate interfaces (and is physically part
+of the GraphQL server, but logically part of ingestion).
+
+#### CollectSub
+
+The collect subcriber service provides a way to express a want for a datasource
+to be used, or indication that more data about a software identifier is desired.
+For example, in parsing an SBOM, if it sees the use of a package with a PURL,
+the ingestor creates and entry to the CollectSub service to indicate more
+information about the PURL is desired (via gRPC).
+
+The collectors all subscribe to this service and will automatically retrieve
+more information about the PURL (or other identifier/datasource) if it knows how
+to handle it. For example, the deps.dev collector would know how to handle
+PURLs, and then retrieve more information about the PURL entries created from
+the ingestor parsing the SBOM.
```

Comment:
i talked to our TW about this, she recommended deduplicating this and moving it out, so let's leave it out of this document

Reply:
make sense...we were expecting to externalize the common content out of this doc and Compose.md.